src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Controllers/ApprenticeController.cs

```diff
@@ -573,13 +573,13 @@ public IActionResult SetCourseForEdit(SelectCourseViewModel model)
         public async Task<IActionResult> SelectDeliveryModelForEdit(EditApprenticeshipRequest request)
         {
             var draft = TempData.GetButDontRemove<EditApprenticeshipRequestViewModel>(ViewModelForEdit);
-            var model = await _modelMapper.Map<Models.SelectDeliveryModelViewModel>(draft);
+            var model = await _modelMapper.Map<EditApprenticeshipDeliveryModelViewModel>(draft);
 
-            if (model.DeliveryModels.Length > 1)
```

Comment:
Not sure if this is the case here but I've been told in the past that "length" is often quicker to process than "count". Length is a property of the object rather than a method. So it's milliseconds faster to run. Not much in the scheme of things but it can add up in a whole project.